app/js/eventteam.js

```diff
@@ -152,13 +161,21 @@ angular.module("teamform-eventteam-app", ["firebase", "ngMaterial"])
     // sort teams by the number of skills matched
     $scope.sortSkillsMatch = function(teams) {
         console.log("sortSkillsMatch()");
-
         return teams.sort(function(a, b) {
             var x = a.skillsMatch.number; var y = b.skillsMatch.number;
             return ((x < y) ? 1 : ((x > y) ? -1 : 0));
         });
     };
 
+    // sort teams by the number of skills missing
+    $scope.sortMissingSkillsMatch = function(teams) {
+        console.log("sortMissingSkillsMatch()");
+        return teams.sort(function(a, b) {
+            var x = a.missingSkillsMatch.number; var y = b.missingSkillsMatch.number;
+            return ((x < y) ? 1 : ((x > y) ? -1 : 0));
+        });
+    };
+
     // sort the teams
     $scope.sortTeams = function(sortBy) {
```

Comment:
add the condition and `$scope.sortMissingSkillsMatch(teams)` call after line 193